doc/api/process.md

```diff
@@ -452,7 +452,7 @@ added: v0.1.27
 
 The `process.argv` property returns a array containing the command line
 arguments passed when the Node.js process was launched. The first element will
-be 'node', the second element will be the name of the JavaScript file. The
+be  [`process.execPath()`], the second element will be the name of the JavaScript file. The
```

Comment:
hmm.. that's an interesting question... but I think that's ok. While there is the potential for conflicts because of the missing `.js`, it's not likely to have a serious impact. I guess that someone could come along and create a separate file named `test` after the app was started that would lead to issues but that seems like a relatively safe edge case.